fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/MoreBiomes.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.biomes.v1;
+
+import net.fabricmc.fabric.impl.biomes.BiomeLists;
+import net.minecraft.world.biome.Biome;
+
+public final class MoreBiomes
```

Comment:
I personally dislike this name, but Kit wanted it for the guava conventions (which I disagree with). I think something like `BiomeUtil` or something would work better. Tbh I'm not happy with BiomeUtil either but it's just as bad as MoreBiomes. Perhaps `FabricBiomes` or `BiomesApi`

Reply:
`FabricBiomes`